src/routes/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Books from '../components/Books';
+
+function Home() {
+  return (
+    <div>
+      <Books />
+    </div>
```

Comment:
![image](https://github.com/unorjikingsley/React-Bookstore/assets/93039359/fd022ee1-e0d7-496d-8ab6-6a75d3d1a810)

- [ ] Kindly have a look at your app because I am not able to add or remove a book in your app 😨 Kindly fix this issue to make your app work as expected 👍